bash/drive-utils.bash

```diff
@@ -151,8 +173,12 @@ _generate_upload_link() {
         -D - || :)" && _clear_line 1 1>&2
     _clear_line 1 1>&2
 
-    uploadlink="$(read -r firstline <<< "${uploadlink/*[L,l]ocation: /}" && printf "%s\n" "${firstline//$'\r'/}")"
-    { [[ -n ${uploadlink} ]] && return 0; } || return 1
+    case "${uploadlink}" in
+        *'ocation: '*'upload_id'*) uploadlink="$(read -r firstline <<< "${uploadlink/*[L,l]ocation: /}" && printf "%s\n" "${firstline//$'\r'/}")" && return 0 ;;
+        *'"message": "Invalid Credentials"'* | * | '') return 1 ;;
```

Comment:
Codacy found an issue: [This pattern never matches because of a previous pattern on line 178.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)